source/deployment/sso-saml-okta.rst

```diff
@@ -88,11 +94,11 @@ Alternatively you can enter the following fields manually:
 
 4. Configure Mattermost to sign SAML requests using the Service Provider Private Key.
 
-5. Set attributes for the SAML Assertions, which will be used to update user information in Mattermost. Attributes for email and username are required and should match the values you entered in Okta earlier. See :ref:`documentation on SAML configuration settings <saml-enterprise>` for more detail.
-
-For Mattermost servers running 3.3 and earlier, the first name and last name attributes are also required fields.
-
-	.. image:: ../../source/images/okta_13_mattermost_attributes.PNG
+5. Set attributes for the SAML Assertions used to update user information in Mattermost. 
+	- Attributes for Email, Username, and Id are required and should match the values you entered in Okta earlier.
+	- For Mattermost servers running 3.3 and earlier, the first name and last name attributes are also required fields.
+	
+	.. image:: ../../source/images/okta_13_mattermost_attributes.png	
```

Comment:
This image isn't displaying in preview.

Reply:
There were 2 files with identical file names (except that one extension was PNG and the other was png). I've removed the older file as part of this PR.

Reply:
Confirmed that the correct image is displaying now.